bower.json

```diff
@@ -9,14 +9,11 @@
     "ember-qunit": "0.4.16",
     "ember-qunit-notifications": "0.1.0",
     "ember-resolver": "~0.1.20",
-    "jquery": "^1.11.3",
+    "jquery": "1.11.3",
```

Comment:
I do think we should keep this in lock-step with ember releases.  I can create an issue and do in a separate PR